scripts/split-e2e-tests.ts

```diff
@@ -7,18 +7,23 @@ type TestRegion = {
   name: string;
   optIn: boolean;
   cognitoSupported: boolean;
+  betaLayerDeployed: boolean; // is the beta layer deployed in this region
```

Comment:
nit: The comment is redundant with the well-named variable :)

```suggestion
  betaLayerDeployed: boolean;
```